includes/discovery/sensors/pre-cache/ciena-sds.inc.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2016 Neil Lathwood <neil@lathwood.co.uk>
```

Comment:
Please use the copyright block from our docs: https://docs.librenms.org/General/Contributing/#copyright